spring-cloud-stream-app-generator/pom.xml

```diff
@@ -17,11 +17,11 @@
 			<plugin>
 				<groupId>org.springframework.cloud.stream.app.plugin</groupId>
 				<artifactId>spring-cloud-stream-app-maven-plugin</artifactId>
-				<version>${scs-app-maven-plugin.version}</version>
+				<version>1.0.0.BUILD-SNAPSHOT</version>
 				<configuration>
 					<javaVersion>1.7</javaVersion>
 					<generatedProjectHome>${session.executionRootDirectory}/apps</generatedProjectHome>
-					<generatedProjectVersion>${project.version}</generatedProjectVersion>
+					<generatedProjectVersion>1.0.0.M1</generatedProjectVersion>
```

Comment:
I believe this is supposed to use `project.version` instead of hard coded value here.